keycloak/keycloak-admin-ui
# Updated subheadings

## Motivation
Updates as per  #1893

## Verification Steps
1. Navigate to each page and verify that subheadings are updated.

Clients subheading text -
Clients are applications and services that can request authentication of a user.

Client scopes subheading text
Client scopes are a common set of protocol mappers and roles that are shared between multiple clients.

Realm roles subheading text
Realm roles are the roles that you define for use in the current realm.

Users subheading text
Users are the users in the current realm.

Groups subheading text
A group is a set of attributes and role mappings that can be applied to a user. You can create, edit, and delete groups and manage their child-parent organization.

Sessions subheading text -
Sessions are sessions of users in this realm and the clients that they access within the session.

Events subheading text
Events are records of user and admin events in this realm. To configure the tracking of these events, go to Event configs (keep the link currently used for Events config).

Realm settings subheading text
Realm settings are settings that control the options for users, applications, roles, and groups in the current realm.

Authentication subheading text
Authentication is the area where you can configure and manage different credential types.

Identity providers subheading text
Identity providers are social networks or identity brokers that allow users to authenticate to Keycloak.

User federation subheading text
User federation provides access to external databases and directories, such as LDAP and Active Directory.

## Checklist:
